src/vulkan-renderer/wrapper/fence.cpp

```diff
@@ -30,7 +30,7 @@ Fence::Fence(const wrapper::Device &device, const std::string &name, const bool
 }
 
 Fence::Fence(Fence &&other) noexcept
-    : m_device(other.m_device), m_fence(std::exchange(other.m_fence, nullptr)), m_name(std::move(other.m_name)) {}
+    : m_device(other.m_device), m_fence(std::exchange(other.m_fence, nullptr)), m_name(other.m_name) {}
```

Comment:
I would probably keep the `std::move` and remove the `const` from `m_name`, because although this change fixes the warning, it means every move of the wrapper copies the name string instead of moving it.

Reply:
I will undo this.

Reply:
Done.